### PR TITLE
Make it possible to have multiline statements in config

### LINKED ISSUE #9404: Allow splitting ERT Config files into multiple lines
**Is your feature request related to a problem? Please describe.**
In the Ert config file, you cannot split a long command line into several lines, like:
 
```python
FORWARD_MODEL SOME_FM(arg1=arg1, arg2=arg2, arg3=arg3, ...)
``` 

 ```python
FORWARD_MODEL SOME_FM(arg1=arg1,
                   arg2=arg2,
                   arg3=arg3,
                   ...
                  )
```
This would improve the readability of long config files. 

**Describe the solution you'd like**
The ability to split the config file into multiple lines to improve readability.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by eivindjahren**
**Issue**
Resolves #9404 

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'pytest tests/ert/unit_tests -n logical -m "not integration_test"'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: docs/ert/reference/configuration/syntax.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+The ert config file, workflows, workflow job configuration files and forward
+model configuration files all share the same syntax. These are lines starting
+with one keyword followed by arguments separated by whitespace. e.g.
+
+::
+
+        QUEUE_SYSTEM LOCAL
+        NUM_REALIZATIONS 12
+        -- A comment
```
Comment (sondreso) [inline docs/ert/reference/configuration/syntax.rst:12]: Maybe add to the text that -- is used for comments as well?

File: docs/ert/reference/configuration/syntax.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        FORWARD_MODEL DELETE_FILE(<FILES>="file1 file-- file3")
+
+        FORWARD_MODEL TEMPLATE_RENDER( \
+            <INPUT_FILES>=parameters.json, \
+            <TEMPLATE_FILE>=<CONFIG_PATH>/resources/SPE1.DATA.jinja2, \
+            <OUTPUT_FILE>=SPE1.DATA \
+        )
+
+As can be seen, newline can be used inside an argument list by escaping with :code:`\` (like with bash). Arguments
```
Comment (sondreso) [inline docs/ert/reference/configuration/syntax.rst:21]: Think we should also add a warning about whitespace after `/`

**Review by sondreso — APPROVED** [review]
Beyond the two minor comments about the documentation I think this looks good!

File: docs/ert/reference/configuration/syntax.rst (hunk 1 above)
Comment (eivindjahren) [reply docs/ert/reference/configuration/syntax.rst:12]: Added.

File: docs/ert/reference/configuration/syntax.rst (hunk 2 above)
Comment (eivindjahren) [reply docs/ert/reference/configuration/syntax.rst]: Added

**Pull request merged**